conf/iot.py

```diff
@@ -20,3 +20,4 @@
 IOT_THING_INTEGRATION = ENVIRONMENT in ('PROD', 'DEMO', 'STAGE', 'DEV')
 IOT_AWS_HOST = os.environ.get('IOT_AWS_HOST', None)
 IOT_GATEWAY_STAGE = ENVIRONMENT.lower()
+IOT_DEVICES_MAX_RESULTS = 10
```

Comment:
What is this for? We can't assume a max number of devices for an organization. Is for pagination?

Reply:
This setting is for the aws IoT module which requires a `maxResults` param for  the number of devices to pull by call and is linked to the recursion done in `get_list_owner_devices`. If we set this to for example 1000 we probably won't need the recursion but the request will take long to fulfill.
For now I opted to keep it consistent with the page pagination size of 10

Reply:
Okay, I see. So this is the page size used for the call to API Gateway. I agree - it makes sense to keep this consistent with the other pagination setting. Can you rename the var to IOT_DEVICES_PAGE_SIZE or something similar?